release/models/openconfig-extensions.yang

```diff
@@ -164,6 +170,21 @@ module openconfig-extensions {
       optimisations for sending or storing the corresponding data.";
   }
 
+  extension telemetry-atomic-exempt {
+    description
+      "The telemetry-atomic-exempt annotation is specified in the context
+      of a subtree (container, or list), and indicates that all nodes
+      within the subtree are not always updated together within the data
+      model of the parent tree. All elements under the
+      subtree may not be updated as a result of a new alarm being raised,
+      or the arrival of a new protocol message that updates the parent
+      tree.
+
+      This annotation allows parent tree containers with telemetry-atomic
+      annotation to not be updated when a more frequently updated subtree.
+      For example, a counters container is present.";
+  }
```

Comment:
It should probably be noted that this extension is not to be used unless there is a parent that already contains `telemetry-atomic`

And on `telemetry-atomic`, while I don't believe we have a case today I believe the interpretation is a blanket "every child of where the annotation exists" is to be sent atomically.  If defined at some parent and as child subtrees grow, this could cause revisiting intentions (in other forums there have been similar discussions on just delivering the content of 1 level depth, etc...)

Other than that, I think this is a fine approach

Minor nits: alignment seems off at least in the UI diff rendering, sentence on L# 183-184 I'm not sure I'm grokking as-is...

Reply:
Thanks @earies 
I have updated the documentation to clarify that the parent should have `telemetry-atomic` extension

Not sure about the alignment issue. Let me know if you still see that.

Reply:
:+1:  - alignment looks fine now

Reply:
Great! Let me know if you have any other feedback as well. Thanks!